scripts/build/Platform/Linux/build_config.json

```diff
@@ -393,7 +393,7 @@
     "COMMAND": "deploy_cdk_applications.sh",
     "PARAMETERS": {
       "NVM_VERSION": "v0.39.1",
-      "CDK_VERSION": "1.179.0",
+      "CDK_VERSION": "1.183.0",
```

Comment:
IMHO we should let these float and allow the python applications to decide (https://peps.python.org/pep-0440/#compatible-release). 

We should be usng pyenv to reduce impact between tests.

Tying to a fixed version is just brittle.

Reply:
Totally agree we should stop pinning to a version, however I wanted to get this project name formulation change out in isolation first, and we know it will fail on the current version so it is bumped here. Would prefer to change the logic that sets and uses the `CDK_VERSION` in a separate PR so it's easier to isolate what's wrong if problems arise.

Reply:
Note, the versions denoted here are tied to the NPM CDK toolkit, not the Python package. 

It's fine to let the version float, though it also means NodeJS has to be maintained as well, as the latest version of the toolkit does not guarantee backwards compatibility with older versions of NodeJS. NVM acts like pyenv in this case for version pinning

Reply:
Thanks for pointing this out @amzn-changml , we'll need to keep that in mind during subsequent changes to how to how we version these.